components/brave_wallet/renderer/js_solana_provider.cc

```diff
@@ -570,8 +564,14 @@ void JSSolanaProvider::OnConnect(
   v8::Local<v8::Context> context = global_context.Get(isolate);
   v8::Context::Scope context_scope(context);
   v8::Local<v8::Value> result;
+  v8::Local<v8::Value> v8_public_key;
   if (error == mojom::SolanaProviderError::kSuccess) {
-    result = CreatePublicKey(context, public_key);
+    v8_public_key = CreatePublicKey(context, public_key);
+    v8::Local<v8::Object> object = v8::Object::New(isolate);
+    CHECK(CreateDataProperty(context, object, u"publicKey", v8_public_key)
```

Comment:
do we have a const for `publicKey` somewhere?

Reply:
also used https://github.com/brave/brave-core/pull/14460/files#diff-7cdd1da2bab576a68d9a1469b45add340ad978aafbdff369f3e590b1aaf8febbR740

Reply:
fixed in 4d3458125b59d884a72a96565a2c43311c31eba1